### ISSUE TITLE
🐛 Assessment tracker a11y Jest test times out

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#11940]
## 🐛 Bug

The assessment tracker a11y Jest test often times out and causes CI to fail.

## 📸 Screenshot

![image](https://github.com/user-attachments/assets/cbd0ad2a-6645-4c9b-91ee-c959752bf9e1)

## 🙋‍♀️ Proposed Solution

Move the test to Playwright.  This will give it more time and it will actually be able to render Hydrogen, too.

## ✅ Acceptance Criteria

- [ ] test does not timeout
- [ ] component is still checked for a11y
